<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuWIndow</class>
 <widget class="QMainWindow" name="MainMenuWIndow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>program_title</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/icon.png</normaloff>icon/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="menu1_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>94</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>File Encryption</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>14</y>
      <width>481</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>program_title</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="menu2_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>144</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>File Decryption</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu3_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>194</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Program Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="sub_title_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>program_sub_title</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="githubAction"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuInfo"/>
  </widget>
  <action name="aboutAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon/info.jpg</normaloff>icon/info.jpg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="githubAction">
   <property name="icon">
    <iconset>
     <normaloff>icon/github.png</normaloff>icon/github.png</iconset>
   </property>
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
